### PR TITLE
Set up To-Do-List with Webpack.

### PR DESCRIPTION
**Pull request by jorgegoco**
-Created the basic structure of the website.
-Set up with Webpack
-Fixed linter errors.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    task.appendChild(taskText);
+    div.appendChild(task);
+    const ellipsis = document.createElement('i');
+    ellipsis.className = 'fa fa-ellipsis-v';
+    div.appendChild(ellipsis);
+    items.appendChild(div);
+  }
+};
+
+populate();
```
Comment (mckent05) [inline src/index.js]: - It is advisable to call this function using an event listener, I suggest you use the `DOMContentLoaded` event listener. Please fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @jorgegoco,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your code is clean and easy to understand 👍 
- All linter checks are passing ✔️ ✔️ 
- Descriptive PR summary and title 👏 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (jorgegoco) [reply src/index.js]: Thank you.

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @jorgegoco    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**